docs/design/DB/dokuha-roadmap.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE users (
+	id UUID DEFAULT gen_random_uuid() PRIMARY KEY,
+	nickname VARCHAR(255) NOT NULL,
+	email VARCHAR(255) UNIQUE NOT NULL,
+	password VARCHAR(255) NOT NULL,
```

Comment:
パスワードには、ハッシュ値などまだ何文字のものが入るかわからないので、TEXTにしたいでふ
```suggestion
	password TEXT NOT NULL,
```